src/fides/api/models/property.py

```diff
@@ -111,6 +146,11 @@ def update(self, db: Session, *, data: Dict[str, Any]) -> Property:
                 f'The path(s) \'{", ".join([matching_path.path for matching_path in matching_paths])}\' are already associated with another property.'
             )
 
+        # Ensure that there is at least 1 default. Relevent if we somehow reach a state where there is no defualt property
+        has_default_property = Property.get_by(db=db, field="is_default", value=True)
+        if not has_default_property:
+            data["is_default"] = True
```

Comment:
There's probably an edge case here where someone is trying to update their sole existing property to have a value of is_default = False which would be allowed -

Reply:
you're right- right now the user has no way to configure a property as default / not default, but I'll prevent them from configuring this and raise an error.

I'll add a test case for this too.